client/apps/plugin-status/view.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLegend from 'components/forms/form-legend';
+import GlobalNotices from 'components/global-notices';
+import HealthView from './health';
+import LogView from './log';
+import notices from 'notices';
+import ServicesStatusView from './services';
+import SettingsGroupCard from 'components/settings-group-card';
+import Toggle from 'components/toggle';
+import {
+	toggleLogging,
+	toggleDebug,
+	save,
+} from './state/actions';
+
+const StatusView = ( { actions, loggingEnabled, debugEnabled, translate } ) => {
+	return (
+		<div>
+			<GlobalNotices id="notices" notices={ notices.list } />
+			<HealthView />
+			<ServicesStatusView />
+			<SettingsGroupCard heading={ translate( 'Debug' ) }>
+				<Toggle
+					id="wcs-toggle-logging"
+					checked={ loggingEnabled }
+					title={ translate( 'Logging' ) }
+					description={ translate( 'Write diagnostic messages to log files. Helpful when contacting support.' ) }
+					trueText={ translate( 'Enabled' ) }
+					falseText={ translate( 'Disabled' ) }
+					saveOnToggle={ true }
+					saveForm={ actions.save }
+					updateValue={ actions.toggleLogging }
+				/>
+				<Toggle
+					id="wcs-toggle-debug"
+					checked={ debugEnabled }
+					title={ translate( 'Debug' ) }
+					description={ translate( 'Display troubleshooting information on the Cart and Checkout pages.' ) }
+					trueText={ translate( 'Enabled' ) }
+					falseText={ translate( 'Disabled' ) }
+					saveOnToggle={ true }
+					saveForm={ actions.save }
+					updateValue={ actions.toggleDebug }
+				/>
+				<LogView
+					logKey="shipping"
+					title={ translate( 'Shipping Log' ) } />
+				<LogView
+					logKey="taxes"
+					title={ translate( 'Taxes Log' ) } />
+				<LogView
+					logKey="other"
+					title={ translate( 'Other Log' ) } />
+			</SettingsGroupCard>
+			<SettingsGroupCard heading={ translate( 'Support' ) }>
+				<FormFieldset>
+					<FormLegend>{ translate( 'Need help?' ) }</FormLegend>
+					<p className="plugin-status__help-description">
+						{ translate( 'Our team is here for you. View our {{docsA}}support docs{{/docsA}} ' +
+						'or {{ticketA}}open a support ticket{{/ticketA}}.', {
+							components: {
+								docsA: <a href="https://docs.woocommerce.com/document/woocommerce-services/" />,
+								ticketA: <a href="https://woocommerce.com/my-account/create-a-ticket/" />,
```

Comment:
Since these links take the user out of his site, do we want to make them `target="_blank"`?